tests/device_test.py

```diff
@@ -551,7 +551,7 @@ def test_commands(self, mock_send_command):
                    "blue", "red", "green", "play", "stop", "pause", "rewind", "forward", "prev", "next", "replay",
                    "advance",
                    "angle", "top_menu", "pop_up_menu", "eject", "karaoke", "netflix", "mode_3d", "zoom_in", "zoom_out",
-                   "browser_back", "browser_forward", "browser_bookmark_list", "list"]
+                   "browser_back", "browser_forward", "browser_bookmark_list", "list", "volume_up", "volume_vown" , "Mute"]
```

Comment:
Can "Mute" be lower case as the other commands are?

Reply:
Just add in elsif because my command id Mute and my function is mute_volume :/

Reply:
Can the method be named mute instead of adding the else if?
I wanted to get of rid of the case for the 3d mode as well.